academic-services/graph/samples-azure-databricks.md

```diff
@@ -0,0 +1,149 @@
+---
+title: PySpark samples for Microsoft Academic Graph
+description: Perform analytics for Microsoft Academic Graph using PySpark on Azure Databricks
+services: microsoft-academic-services
+ms.topic: tutorial
+ms.service: microsoft-academic-services
+ms.date: 3/26/2019
+---
+# PySpark analytics samples for Microsoft Academic Graph
+
+Illustrates how to perform analytics for Microsoft Academic Graph using PySpark on Azure Databricks.
+
+## Sample projects
+
+* [Extract Affiliation ID for an Affiliation](https://github.com/Azure-Samples/microsoft-academic-graph-pyspark-samples/blob/master/src/Lab1_ExtractAffiliation.py)
+* [Join Conferences and Journals as Venues](https://github.com/Azure-Samples/microsoft-academic-graph-pyspark-samples/blob/master/src/Lab2_UnionVenues.py)
+* [Get publications from an Affiliation](https://github.com/Azure-Samples/microsoft-academic-graph-pyspark-samples/blob/master/src/Lab3_JoinPaperAuthorAffiliation.py)
+* [Get authors from an Affiliation and the publication details](https://github.com/Azure-Samples/microsoft-academic-graph-pyspark-samples/blob/master/src/Lab4_CreateTable_Extract.py)
+* [Get Field-Of-Studies for an Affiliation](https://github.com/Azure-Samples/microsoft-academic-graph-pyspark-samples/blob/master/src/Lab5_CreateTableByTvf.py)
```

Comment:
Shouldn't this simply be "Fields of Study"?

Reply:
fixed